src/main/resources/com/mysql/cj/LocalizedErrorMessages.properties

```diff
@@ -1055,7 +1055,8 @@ ConnectionProperties.useAwsIam=Set to true to use AWS IAM database authenticatio
 
 AuroraTopologyService.1=[AuroraTopologyService] clusterId=''{0}''
 AuroraTopologyService.2=[AuroraTopologyService] clusterInstance host=''{0}'', port={1,number,#}, database=''{2}''
-AuroraTopologyService.3=[AuroraTopologyService] The topology query returned an invalid topology - no writer instance detected
+AuroraTopologyService.3=[AuroraTopologyService] The topology query returned an invalid topology - no writer instance detected.
+AuroraTopologyService.4=[AuroraTopologyService] The topology query returned an invalid topology - multi-writer is not supported.
```

Comment:
That looks like a message we no longer need. Everything related to multi-writers should go.